models/marts/dag/fct_hard_coded_references.sql

```diff
@@ -11,7 +11,7 @@ final as (
         resource_name as model,
         hard_coded_references
     from models
-    where hard_coded_references is not null
+    where hard_coded_references != ''
```

Comment:
why the change from null to empty string?

Reply:
Because I changed `wrap_string_with_quotes` to handle `none` and `''` separately